### PR TITLE
Add support for subwoofer gain controls

### PR DESCRIPTION
**Pull request by jjlawren**
Adds support for controlling the subwoofer gain using the `SoCo.sub_gain` property. This property name matches the key from the `RenderingControl` event payload.

Needs validation testing from a system that contains a subwoofer.

### CODE REVIEW

File: soco/core.py [hunk 1]
```diff
@@ -1122,6 +1123,43 @@ def sub_enabled(self, enable):
...
+    def sub_gain(self, level):
+        """Set the subwoofer gain level.
+
+        :param level: Desired subwoofer gain level (-10 to 10)
+        :type level: int
+        """
+        if not self.has_subwoofer:
+            raise NotSupportedException("This group does not have a subwoofer")
+
+        if not -10 <= level <= 10:
```
Comment (bachya) [inline soco/core.py:1150]: In my Sonos iOS app, the gain goes from -15 to 15.

**Review by bachya — CHANGES_REQUESTED** [review]
Tested both the setter and getter and they look good: the Sonos app properly reflects the changes in both cases. Need to change the level limits.

**Comment by jjlawren** [comment]
@bachya Thanks for testing, adjusted thresholds based on your feedback. 👍

**Comment by pwt** [comment]
I'll test (and presumably merge) this later today. Sorry for the delay.

**Comment by pwt** [comment]
Working perfectly with my Play:1 x 2 + Sub rig. Also behaving as expected when tested against a device without a Sub. 
Thanks for your work on this -- it's something I've (selfishly) wanted for a while.

I'll merge as-is, but it would be nice to add a test in a subsequent PR.

**Pull request merged**